contrib/cni/README.md

```diff
@@ -6,7 +6,7 @@ basis for your own configurations (distributions should package these files in
 example directories).
 
 To use these configurations, place them in `/etc/cni/net.d` (or the directory
-specified by `crio.network.network_dir` in your `crio.conf`).
+specified by `crio.network.network_dir` in your overwrite of `/etc/crio/crio.conf.d`).
```

Comment:
I'm not a big fan of the word "overwrite".  In other docs, were saying something like "replace the `crio.network.network-dir` stanza in ..."  I'd lean towards that, but I don't have a strong opinion on it.